superset-frontend/src/features/alerts/AlertReportModal.test.tsx

```diff
@@ -526,7 +523,9 @@ test('renders default Schedule fields', async () => {
     useRedux: true,
   });
   userEvent.click(screen.getByTestId('schedule-panel'));
-  await waitForElementToBeRemoved(() => screen.queryByLabelText('Loading'));
+  if (screen.queryByLabelText('Loading')) {
```

Comment:
I think conditionals in tests might not be the preferred approach. What's the scenario?

Reply:
Loading no longer exists so removed this as well